scripts/lp-bugs-to-gh-issues.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+
+from launchpadlib.launchpad import Launchpad
+from github import Github
+
+
+class GithubApi:
+    def __init__(self):
+        self.repo_name = None
+        self.g = None
+
+    def set_env(self, token, repo_name):
+        self.set_token(token)
+        self.set_repository(repo_name)
+
+    def set_token(self, token):
+        self.g = Github(token)
+
+    def set_repository(self, repo_name):
+        self.repo_name = repo_name
+
+    def create_issue(self, title, body):
+        repo = self.g.get_repo(self.repo_name)
+        issue = repo.create_issue(title=title, body=body)
+
+        return issue
+
+    def get_repo_id(self):
+        repo = self.g.get_repo(self.repo_name)
+        self.repo_id = repo.id
+        return repo.id
+
+
+github_token = os.getenv("GITHUB_TOKEN")
+github_repo = "canonical-web-and-design/maas-ui"
+github_session = GithubApi()
+github_session.set_env(github_token, github_repo)
+
+# The repository to add this issue to
+status = ["New"]
+launchpad = Launchpad.login_with("Canonical web team stats", "production")
+project = launchpad.projects["maas"]
+ui_project = launchpad.projects["maas-ui"]
+
+
+def generate_open_bugs():
+    for task in project.searchTasks(status=status, tags=["ui"]):
```

Comment:
You define this variable outside this function, just for use once within this function. It seems less clear. You should just write:

```suggestion
    for task in project.searchTasks(status=["New"], tags=["ui"]):
```

Reply:
Good point, done